### PR TITLE
Homemade Ice Cream Recipe

### PR DESCRIPTION
**Pull request by Socerest2**
- Homemade Ice Cream

### CODE REVIEW

**Review by DGKSK8LIFE — APPROVED** [review]

File: src/homemade-ice-cream.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+_Easy Homemade Ice Cream_
+
+- ⏲️ Prep time: 5 min
+- 🍳 Cook time: 10 min
+- 💤 Rest time: 2-4 hours
+- 🍽️ Servings: 1 Serving
+
+## Ingredients
+
+- A Mason jar or some equivilent
```
Comment (hallzy) [inline src/homemade-ice-cream.md:12]:
```suggestion
- A Mason jar or some equivalent
```

File: src/homemade-ice-cream.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- 1 Cup Of Cream
+- 1½ drops of vanilla extract
+- 1½ Tablespoons of sugar
+- A small pinch of salt
```
Comment (hallzy) [inline src/homemade-ice-cream.md:16]: Salt/Pepper shouldn't be listed as ingredients.

I also noticed that it isn't listed in the directions. If there is a specific place that salt should be added, you can mention it in the directions, but no need for it in ingredients.

File: src/homemade-ice-cream.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## Directions
+
+1. Put all your ingredients in the jar. (Besides your toppings)
+2. Shake your ingrediants up. This takes around 10 minutes, just keep shaking until the ingredients become thickened.
+3. Now add your toppings in
+4. You are done! Now put your jar in your freezer for at least 2 hours, I put mine in for 4 hours. (Remember longer is better!)
+## Contribution
+
+- Vince - [website](https://vincestuff.com), [github](https://github.com/Socerest2)
+;tags: dessert american
```
Comment (hallzy) [inline src/homemade-ice-cream.md:28]:
```suggestion
4. You are done! Now put your jar in your freezer for at least 2 hours, I put mine in for 4 hours. (Remember longer is better!)

## Contribution

- Vince - [website](https://vincestuff.com), [github](https://github.com/Socerest2)

;tags: dessert american
```

Add some extra new lines between, for better separation.

**Review by hallzy — CHANGES_REQUESTED** [review]
Is there an image that you can upload for this recipe?

**Comment by Socerest2** [comment]
> Is there an image that you can upload for this recipe?

Not at the moment since I ran out of heavy cream.

**Review by hallzy — APPROVED** [review]

**Comment by Knarkzel** [comment]
This is a very sugary recipe indeed...

**Review by zyansheep — APPROVED** [review]
wow this looks insanely simple and good. any pics?

**Comment by Socerest2** [comment]
> wow this looks insanely simple and good. any pics?

Thanks! Sadly not yet, but soon, hopefully!

**Comment by Socerest2** [comment]
Any possibility of this being added?

**Comment by zyansheep** [comment]
> Any possibility of this being added?

@LukeSmithxyz What do you think?

**Comment by DGKSK8LIFE** [comment]
> > Any possibility of this being added?
> 
> @LukeSmithxyz What do you think?

He's too busy grilling to merge our PRs atm.

**Comment by Socerest2** [comment]
Too much grillin - No Ice Cream

**Pull request closed without merging**